chargebee_byte/client.py

```diff
@@ -13,3 +14,14 @@ def get_subscriptions(self, parameters=None):
         response = requests.get(self.api_url + request.path, auth=self.auth, params=request.data)
         response.raise_for_status()
         return response.json()
+
+    def get_all_subscriptions(self, parameters=None):
+        ret = self.get_subscriptions(parameters)
+        subscriptions = ret['list']
+
+        if 'next_offset' in ret:
+            new_parameters = deepcopy(parameters) or {}
```

Comment:
Maybe you meant `deepcopy(parameters) if parameters else {}` here?

Reply:
No, this should work I think:
```
In [1]: from copy import deepcopy

In [2]: x = {'henk': 'hoi'}

In [3]: y = deepcopy(x) or {}

In [4]: y
Out[4]: {'henk': 'hoi'}

In [5]: y.update({'henk': 'banaan'})

In [6]: y
Out[6]: {'henk': 'banaan'}

In [7]: x
Out[7]: {'henk': 'hoi'}
```

Reply:
And:
```
In [9]: x = None

In [10]: y = deepcopy(x) or {}

In [11]: y
Out[11]: {}

```

Reply:
Yeah it'll work but it'll always call `deepcopy` even when not needed. Nevermind though, not important